app/src/main/res/layout/fragment_map.xml

```diff
@@ -6,20 +6,35 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent">
 
-        <Button
-            android:id="@+id/map_get_my_location"
-            style="@style/Widget.MaterialComponents.Button.Icon"
-            android:layout_width="55dp"
-            android:layout_height="wrap_content"
-            android:layout_gravity="bottom|end"
-            android:layout_margin="10dp"
-            android:background="@drawable/rounded_corner"
-            app:icon="@android:drawable/ic_menu_mylocation" />
-
 
         <com.google.android.gms.maps.MapView
             android:id="@+id/mapView"
             android:layout_width="match_parent"
             android:layout_height="match_parent" />
+
+        <LinearLayout
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="bottom|end"
+            android:orientation="vertical"
+            android:layout_margin="10dp">
+
+            <Button
+                android:id="@+id/map_get_my_location"
+                style="@style/Widget.MaterialComponents.Button.Icon"
+                android:layout_width="55dp"
+                android:layout_height="wrap_content"
+                android:background="@drawable/rounded_corner"
+                app:icon="@android:drawable/ic_menu_mylocation" />
+
+            <Button
+                android:id="@+id/map_start_search"
+                style="@style/Widget.MaterialComponents.Button.Icon"
+                android:layout_width="55dp"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="10dp"
+                android:background="@drawable/rounded_corner"
+                app:icon="@android:drawable/ic_menu_search" />
```

Comment:
I'd invert these two buttons: usually the "get location" button is the one in the corner